nil/core-concepts/transaction-lifecycle.mdx

```diff
@@ -2,32 +2,33 @@
 
 ## Definition
 
-In =nil;, a transaction passes the following stages before reaching finality.
+In =nil;, a transaction carrying an external message passes the following stages before reaching finality.
```

Comment:
Transactions don't carry external messages. Transactions and messages are synonyms.
Let's just say that "external messages pass the following stages"